sws_extension.rc

```diff
@@ -140,7 +140,8 @@ BEGIN
     CONTROL         "Sends",IDC_SENDS,"Button",BS_AUTOCHECKBOX | WS_TABSTOP,171,91,35,10
     CONTROL         "Vis",IDC_VISIBILITY,"Button",BS_AUTOCHECKBOX | WS_TABSTOP,140,103,27,10
     CONTROL         "Selection",IDC_SELECTION,"Button",BS_AUTOCHECKBOX | WS_TABSTOP,171,103,45,10
-    CONTROL         "Phase", IDC_PHASE, "Button", BS_AUTOCHECKBOX | WS_TABSTOP, 140, 115, 45, 10
+    CONTROL         "Pol", IDC_PHASE, "Button", BS_AUTOCHECKBOX | WS_TABSTOP, 140, 115, 32, 10
```

Comment:
Changing Phase to Pol(arity) although it's still labeled as "Phase" in REAPER and other areas of SWS and Snapshots?

Reply:
Uhm that was purely because Phase didn't quite fit anymore. :)
!(https://i.imgur.com/1YDudqp.jpg)

Any suggestion?

Reply:
There's unused room to the right, the second column can be moved a little bit. The panel could be made a little bigger to accommodate the new contents if needed.

![](https://i.imgur.com/TfQLyya.png)

(There's room for writing "Visibility" in full by moving it in place of the Sends checkbox)

Reply:
> There's enough space for writing "Visibility" in full by moving it in place of the Sends checkbox too

I've done that but on rethought, I hope noone will complain about wasted muscle memory if the tickboxes are swopped?

Reply:
Hmm yeah, and it would not match the checkboxes in the "SWS Snapshot Paste" dialog anymore. I think it's clearer and looks better than the abbreviated form though.

The paste dialog should be updated to match the changes in the main snapshots window. The new Phase feature is missing there by the way.

Reply:
> The paste dialog should be updated to match the changes in the main snapshots window. The new Phase feature is missing there by the way.

Thanks.
![](https://i.imgur.com/5wlRX2n.jpg)